ros2/r2b2/ubuntu/xenial/images.yaml.em

```diff
@@ -0,0 +1,46 @@
+%YAML 1.1
+# ROS2 Dockerfile database
+---
+images:
+    @(ros2distro_name)-core:
+        base_image: @(os_name):@(os_code_name)
+        maintainer_name: @(maintainer_name)
+        template_name: docker_images/create_ros2_core_image.Dockerfile.em
+        entrypoint_name: docker_images/ros2_entrypoint.sh
+        template_packages:
+            - docker_templates
+        packages:
+            - python3-pip
+        pip3_install:
+            - argcomplete
+            - flake8
+            - flake8-import-order
```

Comment:
yes, `flake8-import-order` must have been forgotten when all the "development dependencies" have been removed